calculus/integration-substitution.md

```diff
@@ -41,3 +41,118 @@ Plugging *u* back in gives the final answer:
 \int xe^{x^2} \d{x} = \frac{1}{2}e^{x^2} + c
 \end{equation}
 ```
+
+## Skill Builder
+### Find General Solutions for:
+(1) $y' = (x+2)\ln(x)$
+
+```{solution}
+\begin{equation}
+y = \int (x+2)\ln(x) \, dx
+\end{equation}
+
+Select $u$ and $dv$ to find $du$ and $v$:
+
+\begin{align}
+u &= \ln(x), & dv &= (x+2)\,dx \\
+du &= \frac{1}{x}\,dx, & v &= \frac{x^2}{2} + 2x
+\end{align}
+
+Apply integration by parts: $uv - \int vdu$
+
+\begin{align}
+y &=\left( \frac{x^2}{2} + 2x \right)\ln(x) - \int \left( \frac{x^2}{2} + 2x \right) \frac{1}{x} \, dx \\
+\end{align}
+
+Solve the integral, don't forget +C
+
+\begin{align}
+&= \left( \frac{x^2}{2} + 2x \right)\ln(x) - \frac{x^2}{4} - 2x + C
+\end{align}
+
+```
+
+(2) $y' = x^5\sin(x)$
+
+```{solution}
+
+\begin{align}
+y = \int x^5 \sin(x) \, dx
+\end{align}
```

Comment:
Use `\begin{equation}\end{equation}` for all equations that take up one single line.